### DISCUSSION TITLE
Human lymphoid organ single cell dataset integration (used in tutorial notebook)

### DISCUSSION THREAD

**Discussion by chsher** [BayraktarLab/cell2location#73] [Q&A] [answered]
Hello, thank you for creating this great tool! 

I'd like to add a dataset to your scRNA-seq reference for lymph node but want to make sure my integration process is consistent with yours. Could you post the code used to integrate the three published scRNA-seq datasets?

Also, I noticed the Park (2020) study identified fibroblasts, but I do not see them in your integrated single cell reference data - were these excluded?

**Comment by vitkl** [comment, marked as answer]
Hi @chsher 

Could you please provide me with your email so that I can put you in touch with Hamish King who performed the integration?

In general, the integration was performed according to the procedure described in the methods section here: https://www.biorxiv.org/content/10.1101/2020.11.15.378125v1.full

**Comment by vitkl** [comment]
Since this issue is more about our paper than the cell2location package I will move it to discussions.
